### ISSUE TITLE
visdom interface is blank

### ISSUE THREAD

**Issue by Ligouhi** [zhenglab/IntrinsicHarmony#3]
first，thanks for your excellent work！
When I execute the training code, the visdom interface does not display the result picture and the training loss. it works when I execute the code of dovenet.
could you tell me how to solve this problem?
thanks again
